src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -284,13 +293,46 @@ private static TryCatchStatement composeStreamRpcCallableMethodSampleCode(
   }
 
   private static TryCatchStatement composeStreamClientRpcCallableMethodSampleCode(
-      Method method, TypeNode clientType) {
+      Method method, TypeNode clientType, Map<String, ResourceName> resourceNames) {
+    // TODO(summerji): Add unit tests
+    // TODO(summerji): Add customize escape for @ character in <pre>{@code..}</pre>
+    // If variant method signatures exists, use the first one.
+    List<MethodArgument> arguments =
+        method.methodSignatures().isEmpty()
+            ? Collections.emptyList()
+            : method.methodSignatures().get(0);
+    MethodInvocationExpr onNextRequestMethodExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(createStreamObserverVarExpr(method, false))
+            .setMethodName("onNext")
+            .setArguments(createVariableExpr(REQUEST_VAR_NAME, method.inputType()))
```

Comment:
Same issue here as below, let's create just one request variable and pass it around.

Reply:
For these types of locally-scoped variables, I would prefer using a local String instead, since (a) it is used only in this method, (b) it helps enforce a single point of creation, and (c) a local String won't be more expensive than the global constant.